malaria24/ona/admin.py

```diff
@@ -77,14 +77,37 @@ def email_view(self, request, pk):
         return HttpResponse(email.html_content)
 
 
+class ActorAdminForm(forms.ModelForm):
+    district = forms.ChoiceField()
+
+    def __init__(self, *args, **kwargs):
+        super(ActorAdminForm, self).__init__(*args, **kwargs)
+        self.fields['district'].choices = [('---------', '---------')] + [(
```

Comment:
This needs to be `[('', '---------')]` otherwise you'll be saving `'---------'` as the default value.